tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Summary.java

```diff
@@ -248,6 +258,25 @@ private static boolean autoSnapshotsAreEnabled() {
         }
     }
 
+    private static Calendar getNextSnap() {
+        final Calendar when = Calendar.getInstance(TimeZone.getTimeZone(AUTO_SNAP_TIME_ZONE));
+        when.setLenient(true); // December 32 = January 1
+        final int tomorrow = when.get(Calendar.DAY_OF_YEAR) + 1;
+        when.set(Calendar.DAY_OF_YEAR, tomorrow);
```

Comment:
```suggestion
        when.add(Calendar.DAY_OF_YEAR, 1);
```

Reply:
oh, that's handy! i've learned something new

this "commit suggestion" thing seems new as well, and also handy